src/NUnitFramework/tests/nunit.framework.tests.csproj

```diff
@@ -44,6 +51,8 @@
 
   <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
     <Reference Include="System.Web" />
+    <Reference Include="System.Windows.Forms" />
+    <Reference Include="WindowsBase" />
```

Comment:
These aren't needed for .NET Framework when `<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">` is used and the two Use properties are set to true. Are they needed now?

Reply:
Those are not needed. Don't know why i left them there. Will remove those.

Reply:
Okay. I wasn't sure. Thanks!